podigee/job/__main__.py

```diff
@@ -236,13 +294,35 @@ def transform_podigee_analytics_to_metrics(analytics_data):
         start_date=date_range.start,
         end_date=date_range.end,
     ),
-    # Podcast metrics - analytics data for the podcast
+    # Podcast metrics like apps and platforms and downloads per day of last 30 days
+    FetchParams(
+        openpodcast_endpoint="metrics",
+        podigee_call=lambda: transform_podigee_analytics_to_metrics(
+            podigee.podcast_analytics(PODCAST_ID, start=date_range.start, end=date_range.end),
+            # we fetch this just every week on Monday and the first day of the month
+            # daily downloads are stored every day
+            not (TODAY_DATE.weekday() == 0 or TODAY_DATE.day == 1)
```

Comment:
Is it really day 1 or should it be day 0?

Reply:
monday is 0